dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/Engine.scala

```diff
@@ -120,6 +136,8 @@ class Engine(context: EngineContext) {
 
 object Engine {
 
+  private val logger: Logger = LoggerFactory.getLogger(Engine.getClass)
```

Comment:
This is unused in `object Engine` is'nt it?

Reply:
Correct, let me remove it.